src/pages/Wallet/SendPage.tsx

```diff
@@ -42,48 +45,34 @@ const SendPage = () => {
 
   const [address, setAddress] = useState('')
   const [amount, setAmount] = useState('')
-  const [addressError, setAddressError] = useState('')
-  const [isChecking, setIsChecking] = useState(false)
   const [isSending, setIsSending] = useState(false)
-
-  const onCloseButtonClick = (isChecking: boolean) => {
-    if (!isChecking) {
-      onModalClose()
-    } else {
-      setIsChecking(false)
-      setOnModalClose(() => () => onCloseButtonClick(false))
-      setModalTitle('Send')
-    }
+  const [step, setStep] = useState<StepIndex>(1)
+
+  useEffect(() => {
+    setModalTitle('Send')
```

Comment:
Not sure if this is useful?

Reply:
I'll do some refactoring and probably get rid of this. Atm it sets the initial title of the modal when the component loads for the first time. I'll refactor the `goToStep` so that this won't be needed.

Reply:
Fixed